src/handlers/swap_handlers.ts

```diff
@@ -197,10 +198,10 @@ export class SwapHandlers {
 
     public async getMarketDepthAsync(req: express.Request, res: express.Response): Promise<void> {
         // NOTE: Internally all ETH trades are for WETH, we just wrap/unwrap automatically
-        const buyTokenSymbolOrAddress = isETHSymbolOrAddress(req.query.buyToken as string)
+        const buyTokenSymbolOrAddress = isNativeSymbolOrAddress(req.query.buyToken as string)
             ? 'WETH'
             : (req.query.buyToken as string);
-        const sellTokenSymbolOrAddress = isETHSymbolOrAddress(req.query.sellToken as string)
+        const sellTokenSymbolOrAddress = isNativeSymbolOrAddress(req.query.sellToken as string)
             ? 'WETH'
```

Comment:
NIT: shouldn't this be `NATIVE_WRAPPED_TOKEN_SYMBOL` instead of `WETH`? If we wanna support market depth for other chains in the future.